CHANGELOG.md

```diff
@@ -1,5 +1,54 @@
 # Change Log
 
+## [0.8.0](https://github.com/confluentinc/ksql/releases/tag/v0.8.0-ksqldb) (2020-03-18)
+
+### Features
+
+* add ARRAY_LENGTH UDF ([#4725](https://github.com/confluentinc/ksql/pull/4725)) ([31a9d9d](https://github.com/confluentinc/ksql/commit/31a9d9d))
+* add confluent-hub to ksqlDB docker image ([#4729](https://github.com/confluentinc/ksql/pull/4729)) ([b74867a](https://github.com/confluentinc/ksql/commit/b74867a))
+* basic auth for new API server ([#4685](https://github.com/confluentinc/ksql/pull/4685)) ([5dcea24](https://github.com/confluentinc/ksql/commit/5dcea24))
+* Integrate new API with existing security framework ([#4713](https://github.com/confluentinc/ksql/pull/4713)) ([0f7de31](https://github.com/confluentinc/ksql/commit/0f7de31))
+* add the topic name to deserialization errors ([#4573](https://github.com/confluentinc/ksql/pull/4573)) ([0f7edf6](https://github.com/confluentinc/ksql/commit/0f7edf6))
+* display errors in CLI in red text ([#4509](https://github.com/confluentinc/ksql/pull/4509)) ([56f9c9b](https://github.com/confluentinc/ksql/commit/56f9c9b))
+* enhance `PRINT TOPIC`'s format detection ([#4551](https://github.com/confluentinc/ksql/pull/4551)) ([8b19bc6](https://github.com/confluentinc/ksql/commit/8b19bc6))
+* Integrate pull queries in new API([#4498](https://github.com/confluentinc/ksql/pull/4498)) ([cd7c058](https://github.com/confluentinc/ksql/commit/cd7c058))
+* introduce JSON_SR format ([#4596](https://github.com/confluentinc/ksql/pull/4596)) ([daa04d2](https://github.com/confluentinc/ksql/commit/daa04d2))
+* log groupby errors to processing logger ([#4575](https://github.com/confluentinc/ksql/pull/4575)) ([b503d25](https://github.com/confluentinc/ksql/commit/b503d25))
+* Plug insert streams into backend ([#4512](https://github.com/confluentinc/ksql/pull/4512)) ([2482f0f](https://github.com/confluentinc/ksql/commit/2482f0f))
```

Comment:
There's a lot in here about the new API server.  I don't think they're meaningful to users. They're more sub-tasks of the feature, where the feature is a new API server. 

I'd suggest dropping all of these:

* basic auth for new API server
* Integrate new API with existing security framework
* Plug insert streams into backend
* Provide upper limit on number of push queries
* Simple perf tool for new API

If the new API is ready for release then it should have only a single mention IMHO.

Reply:
Makes sense, I've removed them.